packages/forms-web-app/src/dynamic-forms/section.js

```diff
@@ -99,7 +100,13 @@ class Section {
 				);
 			}
 
-			// move to next question if answer not provided for this question or for file upload questions the length of uploaded files is less than 1
+			if (question.subQuestion instanceof AddressAddMoreQuestion) {
+				missingRequiredFiles = !journeyResponse.answers.SubmissionAddress.some(
+					(address) => address.fieldName === question.subQuestion.fieldName
+				);
```

Comment:
I realise now this could be an `every` with the negative inside the predicate, might be 1% more readable but also don't worry because then the tests will run again